bash/curl/security/curl-eval.yaml

```diff
@@ -21,6 +21,8 @@ rules:
   pattern-sources:
   - pattern-either:
     - pattern: $DATA=$(curl ...)
```

Comment:
I think grayson used `$DATA=$(curl ...)` and `eval curl ...` to avoid false positives -- @minusworld probably has more context on this than me though 😅

Reply:
Since this is just updating the test case, I think it's fine for now - I can check to see if Iago's suggestions work as expected at a later time

Reply:
Note that new features have been added in the last two weeks:
* simple expression matching: the pattern `foo` now matches in `cmd foo`. No need for the pattern to be `... foo ...`.
* deep ellipsis: pattern `$DATA=<... $(curl ...) ...>` matches `x="foo $(curl url)"`.